ext/reflection/php_reflection.c

```diff
@@ -305,6 +313,8 @@ static void reflection_free_objects_storage(zend_object *object) /* {{{ */
 		case REF_TYPE_PARAMETER:
 			reference = (parameter_reference*)intern->ptr;
 			_free_function(reference->fptr);
+			/* fallthrough */
+		case REF_TYPE_TYPE:
```

Comment:
Not totally sure here, but shouldn't `_copy_function` be called when invoking the factory and the `_free_function` call be done here? Otherwise the trampoline function might get freed.